go/pkg/credshelper/credshelper.go

```diff
@@ -183,7 +183,7 @@ func (ts *externalTokenSource) Token() (*oauth2.Token, error) {
 func (ts *externalTokenSource) GetRequestMetadata(ctx context.Context, uri ...string) (map[string]string, error) {
 	ts.headersLock.RLock()
 	defer ts.headersLock.RUnlock()
-	if ts.expiry.Before(nowFn()) {
+	if ts.expiry.Before(nowFn().Add(-5*time.Minute)) {
```

Comment:
Can you define it as a constant at the top of the file?

Reply:
Done